lumicks/pylake/correlated_stack.py

```diff
@@ -115,6 +115,20 @@ def crop_by_pixels(self, x_min, x_max, y_min, y_max):
         data = self.src.with_roi(np.array([x_min, x_max, y_min, y_max]))
         return self.from_dataset(data, self.name, self.start_idx, self.stop_idx)
 
+    def define_tether(self, point_1, point_2):
```

Comment:
Nitpick: I think numbering without underscores is a bit more common:
```suggestion
    def define_tether(self, point1, point2):
```